### DISCUSSION TITLE
Any way to get more information on a hard crash/freeze?

### DISCUSSION THREAD

**Discussion by 2jfw** [airsdk/Adobe-Runtime-Support#2124] [Q&A]
We are having a rather strange but reproducible freeze/hard crash with our AIR/Flex app (AIR 33.1.1.795) - is there any way to get something like an internal stack trace or reveal further information to provide this to you @ajwfrost ?

**Comment by ajwfrost** [comment]
Is this on Windows? If there's a process that's stuck, you can right-click on the task bar (or use ctrl-alt-delete) and choose "Task Manager", then if you go to the "Details" tab and find the process/application name, right click on that and you should see "Create dump file".

thanks

**Reply by 2jfw** [reply]
Thanks Andrew, I will send you a link to the memory dump file by email (adobe.support@harman.com)
**EDIT:** Link has been sent to you
